Test/FMI3/parser_test_xmls/model_structure/invalid/fmi2_style_list/modelDescription.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fmiModelDescription
+  fmiVersion="3.0"
+  modelName="myModelName"
+  guid="myGuid"
+  description="myDescription"
+  author="myAuthor"
+  generationTool="myGenerationTool"
+  generationDateAndTime="2002-05-30T09:00:00"
+  version="myModelVersion"
+  variableNamingConvention="structured"
+  numberOfEventIndicators="3">
```

Comment:
I think we should remove all optional attributes to minimize clutter. You can copy from #44. I'm trying to do it for all new XMLs.

Reply:
Fixed. I had simply copy pasted the valid one and then broke it, didn't consider the overhead here.